src/main/java/uk/gov/hmcts/reform/draftstore/service/ScheduledTasks.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.hmcts.reform.draftstore.service;
+
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.draftstore.data.DraftStoreDAO;
+
+@Component
+public class ScheduledTasks {
+
+    private final DraftStoreDAO repo;
+
+    public ScheduledTasks(DraftStoreDAO repo) {
+        this.repo = repo;
+    }
+
+    @Scheduled(cron = "${maxAge.cron}")
```

Comment:
How about `deleteOldDrafts.cron`?

Reply:
it was `cron.defaultOldDrafts` initially, but I wanted to group default max age and cron in config :)